test/integration/platform_admin_can_only_delete_cars_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class PlatformAdminCanOnlyDeleteCarsTest < ActionDispatch::IntegrationTest
+  test 'platform admin can only delete cars' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.